DP02_13a_Image_Cutout_SciDemo.ipynb

```diff
@@ -73,11 +73,11 @@
    "source": [
```

Comment:
<p>You might want to update this to adhere to the (newly merged) guidelines at <a href="https://rtn-045.lsst.io/#tap-queries" target="_blank">https://rtn-045.lsst.io/#tap-queries</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Thanks! Added assert job.phase == 'COMPLETED' which was missing</p>